### ISSUE TITLE
[MU4 Issue] Dynamics Inspector not implemented

### ISSUE THREAD

**Issue by vshalkevich** [musescore/MuseScore#7822]
**Describe the bug**
If you select Dynamics and go to Inspector there is empty space

**To Reproduce**
1. Create a score
2. Add notes
3. Click to Palettes from the instruments panel tab
4. Select Dynamics section
5. Add any Dynamic to a note
6. Select Dynamic
7. Click to Inspector from the instruments panel tab 

**Expected behavior**
Inspector has options for Dynamics

**Screenshots**

**Desktop (please complete the following information):**
MacOS

**Additional context**
Could not reproduce on Windows 10

**Comment by Howard-C** [comment]
I think the required settings are all in `General > Playback > Dynamics`. The "style" and "placement" options in 3.x are likely omitted from 4.0 design (see the section for tempos for example).

**Comment by wizofaus** [comment]
The only "dynamics" settings I can see are after clicking "Playback" then switching to the "Dynamic" sub-tab (the "General" one has nothing on it). But in MU3 you can also change the text style properties (though there are some bugs here, in some cases the "italic" style gets applied to the dynamic symbol itself which looks wrong as it's already styled as italic).
Not sure what the intention is here for MU4 - @Tantacrul ? There's a few types of elements (e.g. glissando, let ring, change instrument) where you can't currently edit the text properties currently - probably relatively straightforward to fix but I have no visibility as to what the intended UX should be.

**Comment by Tantacrul** [comment]
Yes. I need to log bugs about this. Basically, for all instances where there is an option to alter a text style or set the placement of text above / below (in the MS3) inspector, they should appear in the text settings in the MS4 inspector in the 'More' button.

![image](https://user-images.githubusercontent.com/21022311/125207018-05734b80-e282-11eb-9bec-f790de0bb8e2.png)

I do think it may be necessary to consider having the placement (above/below) option in a dedicated section in the inspector for any items where that option makes sense.

Basically - the idea is that if a selected item is text, then any relevant formatting options should be contained in the text panel.

Happy to chat this through in more detail.

**Comment by wizofaus** [comment]
If I could see whatever wireframes/ mock ups/ design docs have been done it would certainly help! But if new dialog layouts etc are needed I'll probably leave it to the core team.

**Comment by wizofaus** [comment]
The problem as reported no longer occurs, but I did notice:
a) the font drop-down is showing blank (it has entries but nothing is selected). In fact for a "pure" dynamic marking (with no additional text) it doesn't really do anything anyway, as the standard dynamic symbols like **_m p f_** are always rendered with the score text font.
b) on trying to change the text style, MuseScore crashed (propertyvalue.h, line 311)

**at** was nullptr
```
>	MuseScore4.exe!mu::engraving::PropertyValue::value<int>() Line 311	C++
 	MuseScore4.exe!mu::engraving::PropertyValue::operator==(const mu::engraving::PropertyValue & v) Line 75	C++
 	MuseScore4.exe!Ms::EngravingObject::undoChangeProperty(Ms::Pid id, const mu::engraving::PropertyValue & v, Ms::PropertyFlags ps) Line 424	C++
 	MuseScore4.exe!Ms::TextBase::undoChangeProperty(Ms::Pid id, const mu::engraving::PropertyValue & v, Ms::PropertyFlags ps) Line 3670	C++
...
```

**Comment by Tantacrul** [comment]
I think that we should disable font and styling options when a dynamic is selected.

In this instance:
![image](https://user-images.githubusercontent.com/21022311/145890892-27b7c2f5-26b9-4dac-ae3d-0858787c24dd.png)

And also while editing the text - as long as **no other text is selected at the same time:**
![image](https://user-images.githubusercontent.com/21022311/145890843-6e9f5a83-8bc7-481e-8a21-ad5fc9583da7.png)

**Comment by wizofaus** [comment]
But if you select a whole dynamic (or any text object) that has a mixture of text and musical symbols, then the options do make sense.
I'd agree that in general if what's selected *can't* be affected by a given control it makes more sense to disable it, as long as it's reasonably obvious why.  But at the moment I don't know that we have much in the way of dynamic control enabling.

**Comment by Tantacrul** [comment]
> But if you select a whole dynamic (or any text object) that has a mixture of text and musical symbols, then the options do make sense.

Agreed.

**Comment by wizofaus** [comment]
Ok - but do we want to keep using this issue, or raise new ones?  Fixing the crash would seem to be P0/P1 - disabling controls that currently don't do anything probably more of a P3 type issue...

**Comment by wizofaus** [comment]
The crash seems to be because the inspector is trying to set the SUB_STYLE property to be a string, but it's actually meant to be an integer.  It's happening for ALL text elements too, will raise if not already.

**Comment by Tantacrul** [comment]
I'd say a crash should be raised separately, yes.

**Comment by RomanPudashkin** [comment]
At the moment, there are options for dynamics in the current master. Remaining issues should be logged separately
